nonlinear_elasticity/nonlinear_elasticity.cc

```diff
@@ -621,12 +621,12 @@ namespace Nonlinear_Elasticity
 
     // Alias: Container, which holds references for all time dependent variables
     // to enable a compact notation
-    state_variables = std::vector({&total_displacement,
-                                   &total_displacement_old,
-                                   &velocity,
-                                   &velocity_old,
-                                   &acceleration,
-                                   &acceleration_old});
+    state_variables = {&total_displacement,
```

Comment:
Alternatively we could increase the C++ standard requirement in `CMakeLists.txt`. My compiler defaults to C++11, but we can definitely increase it at least to C++14 (preCICE requirement). But I am not sure [which one exactly we would need here](https://en.cppreference.com/w/cpp/container/vector/vector).

Does deal.II use such a constructor internally?

Otherwise, this should be fine, I guess.

Reply:
> Does deal.II use such a constructor internally?

I'm not sure. Since this solution works now, let's stick to it. If similar issues come up in the future, we can still require c++14. 

Thanks!